src/Rootfs.jl

```diff
@@ -517,6 +524,12 @@ function gcc_version(p::AbstractPlatform,GCC_builds::Vector{GCCBuild},
 end
 
 function llvm_version(p::AbstractPlatform, LLVM_builds::Vector{LLVMBuild})
+    # # We only have LLVM 18 or newer for RISC-V.
+    # # (This could be changed down to LLVM 6.0.)
+    # if arch(p) == "riscv64"
+    #     LLVM_builds = filter(b -> getversion(b) ≥ v"6", LLVM_builds)
```

Comment:
Should this be uncommented and filter llvm 18+?

Reply:
It seems riscv64 support was introduced in LLVM 3.9 (I didn't test this) so we should be good without this.

Reply:
I thought we had to enable the targets in the LLVM recipe, but I see we have all the targets enabled by default:
```console
sandbox:${WORKSPACE} # /opt/x86_64-linux-musl/bin/clang -print-targets

  Registered Targets:
    aarch64     - AArch64 (little endian)
    aarch64_32  - AArch64 (little endian ILP32)
    aarch64_be  - AArch64 (big endian)
    amdgcn      - AMD GCN GPUs
    arm         - ARM
    arm64       - ARM64 (little endian)
    arm64_32    - ARM64 (little endian ILP32)
    armeb       - ARM (big endian)
    avr         - Atmel AVR Microcontroller
    bpf         - BPF (host endian)
    bpfeb       - BPF (big endian)
    bpfel       - BPF (little endian)
    hexagon     - Hexagon
    lanai       - Lanai
    loongarch32 - 32-bit LoongArch
    loongarch64 - 64-bit LoongArch
    mips        - MIPS (32-bit big endian)
    mips64      - MIPS (64-bit big endian)
    mips64el    - MIPS (64-bit little endian)
    mipsel      - MIPS (32-bit little endian)
    msp430      - MSP430 [experimental]
    nvptx       - NVIDIA PTX 32-bit
    nvptx64     - NVIDIA PTX 64-bit
    ppc32       - PowerPC 32
    ppc32le     - PowerPC 32 LE
    ppc64       - PowerPC 64
    ppc64le     - PowerPC 64 LE
    r600        - AMD GPUs HD2XXX-HD6XXX
    riscv32     - 32-bit RISC-V
    riscv64     - 64-bit RISC-V
    sparc       - Sparc
    sparcel     - Sparc LE
    sparcv9     - Sparc V9
    systemz     - SystemZ
    thumb       - Thumb
    thumbeb     - Thumb (big endian)
    ve          - VE
    wasm32      - WebAssembly 32-bit
    wasm64      - WebAssembly 64-bit
    x86         - 32-bit X86: Pentium-Pro and above
    x86-64      - 64-bit X86: EM64T and AMD64
    xcore       - XCore
```

But then I'm missing why this PR is updating the LLVMBootstrap artifact for llvm 18 (which is why I thought we had to manually enable the targets)

Reply:
That's probably not necessary. I'm not quite sure about the new artifacts – please review them, quite a few might not be necessary.

Reply:
Did you change anything locally relative to llvm? I don't see anything in https://github.com/JuliaPackaging/Yggdrasil/pull/8468

Reply:
I don't think I did. I got confused by GCC (which requires a new build for every architecture) and LLVM (which doesn't).

Reply:
I'd suggest reverting the new LLVMBootstrap shard then.

Reply:
Done.